src/include/mutex.i

```diff
@@ -196,6 +198,7 @@ __wt_spin_init(WT_SESSION_IMPL *session, WT_SPINLOCK *t, const char *name)
 	}
 
 	t->name = name;
+	t->stat_count_off = t->stat_app_usecs_off = t->stat_int_usecs_off = -1;
```

Comment:
While I see this fixes the incorrect initialization, I'll take a pass at making the initialization of the WT structure a unified path rather than have it in (at least) 3 places.